audit/src/test/resources/audit.yml

```diff
@@ -0,0 +1,61 @@
+# AuditHandler will pick some important fields from headers and tokens and logs into a audit appender.
+---
+# Enable Audit
+enabled: true
+
+# Enable mask
+mask: false
+
+# Output response status code
+statusCode: true
+
+# Output response time
+responseTime: true
+
+# audit only on error responses; by default log for any return status code
+# if true, audit at error level if status code >= 400
+# log level is controlled by logLevel
+auditOnError: false
+
+# log level; by default set to info
+logLevelIsError: false
+
+
+# Output header elements. You can add more if you want.
+headers:
+
+# Correlation Id
+- X-Correlation-Id
+
+# Traceability Id
+- X-Traceability-Id
+
+# Output from id token and access token
+audit:
+
+# Client Id
+- client_id
+
+# User Id in id token, this is optional
+- user_id
+
+# Client Id in scope/access token, this is optional
+- scope_client_id
+
+# Request endpoint uri@method.
+- endpoint
+
+# Service ID assigned to the service, this is optional and must be set by the service in its implementation
+- serviceId
+
+# Request Body, this is optional and must be set by the service in its implementation
+- request
+
+# Response payload, this is optional and must be set by the service in its implementation
+- response
+
+# Request query parameters
+- queryParameters
```

Comment:
you added queryParams, what if somebody wants to audit path params and cookies?

Reply:
Thanks for pointing out. 

Since my client now only needs audit query parameters, I was limited to audit query parameters before. I think we can try to cover more audit point.

Reply:
I have realized that serviceId is in the audit list but it is not implemented. It is only mentioned in the comment section at the beginning of the handler. When the audit log aggregated to a central location, the serviceId should be the key info. @jiachen1120 Could you please add this to this PR? It should be retrieved from the server.yml file. Let me know if you have questions or concerns.

Reply:
Sure. Will do